app/admin-tab/processes/process/route.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params /*, transition*/) {
+    var processInstance = this.store.getById('processinstance', params.process_id);
```

Comment:
This normally works because the parent route loads all of them so this one is theoretically already loaded, but will fail if there is no instance with that process_id.  Safer to do:

```
return this.get('store').find('processinstance', params.process_id).then((processInstance) => {
```

and find will resolve quickly (without an api call) if that resource is already in the store.